VMTests/vmRandomTest/201503110226PYTHON_DUP6.json

```diff
@@ -1,51 +0,0 @@
-{
-    "201503110226PYTHON_DUP6" : {
```

Comment:
this test was failing on python someday

Reply:
Is that a reason to keep the test or to remove the test?

Reply:
was a long time ago. but if it contains CALL CREATE SELFDESTRUCT then it probably does not make sense to keep it in VMTests

Reply:
It contains `BALANCE` so it's being removed.

Reply:
the balance is easy to replace with any other opcode or remove it. maybe the issue was in other part of the code.

Reply:
It doesn't matter if `BALANCE` is easy or not.  When a client is running a VM test, the client should contain no world state, so the client should not be able to run BALANCE.

Reply:
So, I'll move this test to GeneralStateTest.

Reply:
ok

Reply:
I added this one as a generalstate test in this PR.